make_queue/templates/make_queue/reservation_actions.html

```diff
@@ -3,7 +3,8 @@
 
 <h3>{% trans "Actions" %}</h3>
 <div class="ui relaxed divided selection action list">
-    <a class="item {% if not can_use or not can_make_more_reservations %}disable{% endif %} enable_popup"
+    {% machine_out_of_order machine as machine_out_of_order %}
+    <a class="item {% if not can_use or machine_out_of_order or not can_make_more_reservations %}disable{% endif %} enable_popup"
```

Comment:
I think something like this would be a bit more readable:
```diff
-    {% machine_out_of_order machine as machine_out_of_order %}
-    <a class="item {% if not can_use or machine_out_of_order or not can_make_more_reservations %}disable{% endif %} enable_popup"
+    <a class="item enable_popup
+              {% if not can_use or machine.get_status == Machine.OUT_OF_ORDER or not can_make_more_reservations %}
+                  disable
+              {% endif %}"
+       href="{% url 'make_reservation' machine=machine %}"
             {% if not user.is_authenticated %}
        data-content="{% trans "You must be logged in to make new reservations" %}"
             {% elif not can_use %}
        data-content="{% get_machine_cannot_use_text machine %}"
             {% elif not can_make_more_reservations %}
        data-content="{% trans "You have reached the maximum number of future reservations" %}"
             {% endif %}
-       href="{% url 'make_reservation' machine=machine %}">
+    >
```
This involves removing the `machine_out_of_order` template tag you created - and yes, I'm aware that there exists a lot of template tags which are in the same style as this one, but, frankly, they're all slightly superfluous, when one can simply use the methods that are part of the model that people are more likely to be familiar with. (E.g. why does the `get_machine_cannot_use_text` template tag exist, when one can simply write `machine.machine_type.cannot_use_text`? It kind of baffles me 😅) And if there is a certain piece of code related to a model object that is commonly written, one could just create a method/property for it on the model itself 😊

Also, my suggested code requires adding this line to [the template context](https://github.com/MAKENTNU/web/blob/b503ec2e53de0ce7027f025075196305cf45a7db/make_queue/views/reservation/calendar.py#L21-L30):
```Python
"Machine": Machine,
```

Reply:
This looks like a much better solution, and yes, i dont totally understand why we use all of the template tags cx

Reply:
Hahah, yeah, and there are so many of them 😅😵

But, yeah, if we want to add more machine statuses that should prevent the user from creating a new reservation (like if the machine is under maintenance, as @Maisergodt mentioned above), it might be a fitting opportunity for creating a model method, like `Machine.is_reservable()`, which could check `get_status()` 😄

Reply:
Ooo good idea. I've already pushed the changes you asked for, but i can do this too and append it ;)